integrations/facebook-pixel/lib/index.js

```diff
@@ -582,6 +583,10 @@ FacebookPixel.prototype.formatTraits = function formatTraits(analytics) {
       .toLowerCase();
   var state = address.state && address.state.toLowerCase();
   var postalCode = address.postalCode;
+  var external_id; // eslint-disable-line
+  if (this.options.keyForExternalId.length) {
```

Comment:
Interesting use of `.length`. `keyForExternalId` is guaranteed to be a string? I think so.

To be safer can do:

I think you can actually just do `if (this.options.keyForExternalId) {`

Should be equivalent to what you have.

Reply:
@mericsson Yes that was a typo, just pushed a commit to resolve.